tensorbay/geometry/box.py

```diff
@@ -217,7 +207,7 @@ def tl(self) -> Vector2D:  # pylint: disable=invalid-name
             The top left point.
 
         """
-        return Vector2D(self._data[:2])
+        return Vector2D(*self._data[:2])
```

Comment:
`Vector2D(*self._data[:2])`
vs
`Vector2D(self._data[0], self._data[1])`

Reply:
`Vector2D(self._data[0], self._data[1])` is faster